src/modules/AutomationModule.cpp

```diff
@@ -53,11 +53,17 @@ void AutomationModule::onLoaded()
 	onDial(0);
 }
 
+void AutomationModule::onShow()
+{
+	// Trigger mLabel update
+	onDial(0);
+}
+
 
 void AutomationModule::onDial(int delta)
 {
 	int track = (static_cast<int>(static_cast<float>(delta) + mParams[0]) + Synth::maxAutomationTracks) % Synth::maxAutomationTracks;
 
 	setParam(0, track);
-	sprintf(mLabel, "Auto%d", track);
+	sprintf(mLabel, "Atm%d", track);
```

Comment:
I think this and the ExtIn label could be fixed with a differently shaped/colored module and having just the input number on it.